tls/pk.h

```diff
@@ -136,7 +138,6 @@ int ttls_pk_decrypt(ttls_pk_context *ctx,
 int ttls_pk_encrypt(ttls_pk_context *ctx,
 		    const unsigned char *input, size_t ilen,
 		    unsigned char *output, size_t *olen, size_t osize);
-int ttls_pk_check_pair(const ttls_pk_context *pub, const ttls_pk_context *prv);
 int ttls_pk_debug(const ttls_pk_context *ctx, ttls_pk_debug_item *items);
```

Comment:
There is no such function anymore, as it was inlined. So this declaration should be removed too.